<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>952</width>
    <height>564</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QChartView" name="waveGraphicsView">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>781</width>
     <height>211</height>
    </rect>
   </property>
  </widget>
  <widget class="QGroupBox" name="waveStyleGroupBox">
   <property name="geometry">
    <rect>
     <x>800</x>
     <y>20</y>
     <width>141</width>
     <height>111</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>117</width>
      <height>47</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="waveStyleVerticalLayout">
     <item>
      <widget class="QRadioButton" name="staticRadioButton">
       <property name="text">
        <string>静态显示</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="dynamicRadioButton">
       <property name="text">
        <string>动态显示</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QPushButton" name="saveDataButton">
   <property name="geometry">
    <rect>
     <x>800</x>
     <y>470</y>
     <width>141</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>存储数据</string>
   </property>
  </widget>
  <widget class="QPushButton" name="readDataButton">
   <property name="geometry">
    <rect>
     <x>800</x>
     <y>510</y>
     <width>141</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>加载数据</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="dataPlainTextEdit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>230</y>
     <width>781</width>
     <height>321</height>
    </rect>
   </property>
  </widget>
  <widget class="QProgressBar" name="procDataProgressBar">
   <property name="geometry">
    <rect>
     <x>800</x>
     <y>410</y>
     <width>141</width>
     <height>23</height>
    </rect>
   </property>
   <property name="value">
    <number>24</number>
   </property>
   <property name="textVisible">
    <bool>false</bool>
   </property>
   <property name="invertedAppearance">
    <bool>false</bool>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QChartView</class>
   <extends>QGraphicsView</extends>
   <header location="global">qchartview.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
